core/src/main/scala/ai/h2o/sparkling/H2OContext.scala

```diff
@@ -244,6 +244,7 @@ class H2OContext private[sparkling] (private val conf: H2OConf) extends H2OConte
           RestApiUtils.shutdownCluster(conf)
         }
       }
+      ProxyStarter.stopFlowProxy()
```

Comment:
What about the cases if a user calls `sc.stop()`? Is proxy going to be shutted down through spark hook?

Reply:
Yup. The stop method is invoked during Spark shutdown hook